src/main/java/org/openrewrite/java/testing/junit5/MigrateFromHamcrest.java

```diff
@@ -0,0 +1,55 @@
+package org.openrewrite.java.testing.junit5;
```

Comment:
The license header is missing. To add it you can either copy it from another source file or run `./gradlew lF`, which will add the header to all files which don't have it yet.